### PR TITLE
Anti-aliasing merriweather

### PR DESCRIPTION
**Pull request by helenemccarron**
Add -webkit-font-smoothing: antialiased; -moz-osx-font-smoothing: grayscale; to Merriweather

### CODE REVIEW

File: resources/styles/global/typography.less [hunk 1]
```diff
@@ -41,6 +41,8 @@
     font-style: normal;
     font-size: @size;
     line-height: @line-height;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
```
Comment (philschatz) [inline resources/styles/global/typography.less]: Is there a non-prefixed rule for this we can include as well?

**Comment by Fredasaurus** [comment]
There is, but it is non-standard and generally not supported.

http://caniuse.com/#feat=font-smooth

File: resources/styles/global/typography.less (hunk 1 above)
Comment (helenemccarron) [inline resources/styles/global/typography.less]: Do you still want it or should we skip it?

Comment (philschatz) [inline resources/styles/global/typography.less]: Does it break if we add it, if not, then probably? It seems like the rest of the vendor-prefixes we use also contain the non-prefixed versions (so browsers *just work* once the rule is officially adopted)

Comment (helenemccarron) [inline resources/styles/global/typography.less]: Right, but in this case, the default value was dropped. I added it, even if it doesn't do anything. It's been removed from the specification and is currently not on the standard track, but who knows?

**Pull request merged**